artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/etc/jolokia-access.xml

```diff
@@ -23,7 +23,7 @@ under the License.
 <restrict>
 
     <cors>
-        <!-- Allow cross origin access from ${http.host} ... -->
+        <!-- Allow cross origin access from ${http.host-for-comment} ... -->
```

Comment:
ncluding in a comment a hostname that doesn't match the real hostname is confusing, what about just removing `${http.host}` from the comment?
```suggestion
        <!-- Allow cross origin access from the origins that match the following pattern -->
```
I

Reply:
That is a much simpler solution if nobody is tied to having the hostname in that comment.

I'll adjust (remove the other changes, and update this comment, then update the PR.

Reply:
Removing the comment altogether is the best solution in my opinion.

Reply:
I've reverted my previous commit, and updated the comment in `jolokia-access.xml` so that it has no substitution.  See what you all think.

Reply:
A test would be good to avoid regressions.
Can you squash your commits and prepend the commit message with `ARTEMIS-4160`, i.e. 
```
ARTEMIS-4160 Handle hostnames with double dashes
```

Reply:
Squished and updated the commit message!